src/components/Layout/Sidebar/SidebarLink.tsx

```diff
@@ -15,7 +15,7 @@ interface SidebarLinkProps {
   selected?: boolean;
   title: string;
   level: number;
-  wip: boolean | undefined;
+  canary: boolean | undefined;
```

Comment:
```suggestion
  canary?: boolean;
```
may be no difference at all 🙈

Reply:
fixed thanks